### DISCUSSION TITLE
Disallowed domains defaults and when to use the setting

### DISCUSSION THREAD

**Discussion by timkgh** [AdguardTeam/AdGuardHome#3588] [Q&A] [answered]
1. What is the story behind the disallowed domains out of the box defaults `version.bind`, `id.server`, `hostname.bind`?

2. If I want to block some domains and all their subdomains, should I use "Disallowed domains" or "Custom filtering rules"?

**Comment by ameshkov** [comment, marked as answer]
> What is the story behind the disallowed domains out of the box defaults version.bind, id.server, hostname.bind?

They are often used by different scanners to check if there's an open resolver.

> If I want to block some domains and all their subdomains, should I use "Disallowed domains" or "Custom filtering rules"?

You should use custom filtering rules.

Disallowed domains purpose is to protect public resolvers from abuse.
